src/com/base/game/inventory/clothing/ClothingType.java

```diff
@@ -1013,6 +1050,54 @@ public String unequipText(GameCharacter clothingOwner, GameCharacter clothingRem
 		}
 	},
 	
+	NECK_COWBELL_COLLAR("a",
+			false,
+			"cowbell collar",
+			"A heavy leather collar, of the type worn by cow.",
+			0,
+			null,
+			InventorySlot.NECK,
+			Rarity.UNCOMMON,
+			null,
+			"neck_cowbell_collar",
+			null,
+			
+			Util.newArrayListOfValues(new ListValue<BlockedParts>(new BlockedParts(DisplacementType.REMOVE_OR_EQUIP, null, null, null))),
+			
+			null,
+			
+			Colour.allMetalColours) {
+		@Override
+		public String equipText(GameCharacter clothingOwner, GameCharacter clothingRemover, boolean rough, AbstractClothing clothing, boolean applyEffects) {
+			if (clothingOwner.isPlayer() && clothingRemover.isPlayer())
+				return "You put on the cowbell collar, fastening the clasp at the back of your neck.";
+			else if (!clothingOwner.isPlayer() && !clothingRemover.isPlayer())
+				return UtilText.genderParsing(clothingOwner,
+						Util.capitaliseSentence(clothingOwner.getName("the")) + " puts a " + clothing.getName(true) + " around <her> neck, reaching around" + " to fasten the clasp at the back.");
```

Comment:
no need to concatenate two string literals (e.g. "a" + "b")  here, you can just turn them into one string literal (e.g. "ab")